### ISSUE TITLE
Cannot find module 'graphql'

### ISSUE THREAD

**Issue by Xample** [yarax/swagger-to-graphql#95]
```tty
sudo npm i -g swagger-to-graphql
swagger-to-graphql --swagger=/swagger-file.json
```
```
MacBook-Pro:~ fvolken$ swagger-to-graphql --swagger=/Volumes/CaseSensitive/ni_front/docs/passtl.json 
module.js:549
    throw err;
    ^

Error: Cannot find module 'graphql'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/swagger-to-graphql/lib/index.js:31:16)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/swagger-to-graphql/bin/swagger2graphql:3:13)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
```

any idea about this 'graphql' module?

**Comment by Xample** [comment]
okay the code is using the global graphql then:
 ```npm -g i graphql``` solved it.
A better option would have had to add it to the package.json

**Comment by Xample** [comment]
I reopen because graphql is actually in the package.json

**Comment by yarax** [comment]
Thank you for the reporting
It was missing graphql dependency (devDependency was there)
fixed in 1.4.3

**Comment by buesing** [comment]
Still getting this.
